doc/Language/statement-prefixes.pod6

```diff
@@ -158,6 +158,7 @@ my $counter;
 my $result = do for ^5 { once $counter = 0; $counter++ };
 say $result; # OUTPUT: «(0 1 2 3 4)␤»
 
+<<<<<<< Updated upstream
```

Comment:
This got left here from a merge, I suppose. :P

Reply:
@2colours. Thank you.